src/index.js

```diff
@@ -227,20 +227,14 @@ Program
   .argument('[type]', 'Subcommands (*if required by query module); available types: ' + String(typeChoices))
   .requiredOption('-a, --args <value>', 'JSON encoded arguments for query (e.g. \'{"get_count": {}}\')')
   .option('-f, --flags <flags>', 'Send additional flags to archwayd by wrapping in a string; e.g. "--height 492520 --limit 10"')
+  .option('-c, --contract <contract_address>', 'Query a specific contract address; e.g "--contract archway1..."')
```

Comment:
Please add validation to this argument as there's in the `metadata` command (lines 171 and 172). The `--contract` arg in `tx` is also missing.